ConverterTests/ConverterTests.csproj

```diff
@@ -1,11 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <Nullable>enable</Nullable>
-
+    <TargetFramework>net48</TargetFramework>

```

Comment:
why older version?

Reply:
@DominikaLos @psarras   As there were warning NU1702. I feel .NET framework should be consistent across project.